kukkuu/tests/conftest.py

```diff
@@ -1,8 +1,42 @@
+import secrets
+
 import pytest
 
 from kukkuu.service import get_hashid_service
+from kukkuu.tests.utils.jwt_utils import generate_symmetric_test_jwt
+from users.factories import UserFactory
 
 
 @pytest.fixture
 def hashids():
     return get_hashid_service()
+
+
+@pytest.fixture(autouse=True)
+def oidc_browser_test_api_token_auth_settings(settings):
+    settings.OIDC_BROWSER_TEST_API_TOKEN_AUTH = {
+        "ENABLED": True,
+        "AUDIENCE": ["kukkuu-api-dev", "profile-api-test", "kukkuu-admin-ui-test"],
+        "API_SCOPE_PREFIX": "",
+        "REQUIRE_API_SCOPE_FOR_AUTHENTICATION": False,
+        "API_AUTHORIZATION_FIELD": "authorization.permissions.scopes",
+        "ISSUER": "https://kukkuu-ui.test.hel.ninja",
+        "AUTH_SCHEME": "Bearer",
+        "USER_RESOLVER": "helusers.oidc.resolve_user",
+        "OIDC_CONFIG_EXPIRATION_TIME": 86400,
+        "JWT_SIGN_SECRET": secrets.token_bytes(32).hex(),
+    }
+
+
+@pytest.fixture
+def get_browser_test_bearer_token_for_user(oidc_browser_test_api_token_auth_settings):
+    """
+    Returns a signed bearer token to authenticate
+    through the authentication made for browser testing.
+    """
+    default_user = UserFactory.build()
+
+    def generate_test_jwt_token(user=default_user):
+        return generate_symmetric_test_jwt(user)
+
+    return generate_test_jwt_token
```

Comment:
Hmm... this returns a function (i.e. `return generate_test_jwt_token` returns the function `generate_test_jwt_token`) rather than the return value of a function call, so the documentation and the code are out-of-sync as the documentation says that this function returns a token, not that it returns a function that returns a token.